examples/integration-tutorials/using-the-gmail-connector/README.md

```diff
@@ -44,15 +74,82 @@ gmail:GmailConfiguration gmailConfig = {
         }
     }
 };
+```
 
-gmail:Client gmailClient = new(gmailConfig);
+With the defined `gmailClient`, a function to send the mail needs to be defined. The sender and recipient of the email is specified and the client is used to send the message. 
+
+```ballerina
+
+function sendEmail(string email) returns http:Response {
+    string messageBody = email;
+    http:Response response = new;
+
+    string userId = "me";
+    gmail:MessageRequest messageRequest = {};
+    messageRequest.recipient = "someone@gmail.com";
+    messageRequest.sender = "somebody@gmail.com";
+    messageRequest.subject = "Gmail Connector test : Payment Status";
+    messageRequest.messageBody = messageBody;
+    messageRequest.contentType = gmail:TEXT_HTML;
+
+    // Send the message.
+    var sendMessageResponse = gmailClient->sendMessage(userId, messageRequest);
+
+    if (sendMessageResponse is (string, string)) {
+        // If successful, print the message ID and thread ID.
+        (string, string) (messageId, threadId) = sendMessageResponse;
+        io:println("Sent Message ID: " + messageId);
```

Comment:
Better to use logging instead of console logging.

```ballerina
import ballerina/log;
log:printError(errorMessage, err = response);
```

Reply:
The console logging is based on the use-case provided in the ballerina docs. The message id and thread id doesnt need to be logged i think. 

Shall i remove them?

Reply:
Ill add the logs. And remove the console logging.

Reply:
Done